### PR TITLE
Initial work to render form submission page.

### PR DESCRIPTION
**Pull request by ahmedhamodi**

### CODE REVIEW

**Comment by ahmedhamodi** [comment]
Ready for review @GodGreg :)

**Comment by ahmedhamodi** [comment]
Not sure why this failed. I'll rebase on master after all your changes and see if that fixes it.

**Comment by ahmedhamodi** [comment]
Now ready for review @GodGreg :tada:

**Review by GodGreg — APPROVED** [review]

File: src/Components/FormCreation/Submission/CreateSubmissionForm.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  const { appUser } = useContext(AuthContext);
+  const [sections, dispatchSectionsUpdate] = useReducer(
+    customFormSectionsReducer,
+    []
+  );
+  const [activeSection, setActiveSection] = useState(0);
+  const [loadForm, refetch] = usePromise(FORM.getForm, {
+    programId: appUser.currentProgram
+  });
+  const [headerData, setHeaderData] = useState({
```
Comment (GodGreg) [inline src/Components/FormCreation/Submission/CreateSubmissionForm.js:26]: We shouldn't need this for formSubmission right?

**Review by GodGreg — APPROVED** [review]

File: src/Components/FormCreation/Submission/CreateSubmissionForm.js (hunk 1 above)
Comment (ahmedhamodi) [reply src/Components/FormCreation/Submission/CreateSubmissionForm.js:26]: That is right, I can remove

**Pull request merged**